src/version.rs

```diff
@@ -246,10 +319,57 @@ impl Version {
         }
     }
 
+    /// Gets the Major version number
+    pub fn major(&self) -> u64 {
+        self.major
+    }
+
+    /// Gets the Minor version number
+    pub fn minor(&self) -> u64 {
+        self.minor
+    }
+
+    /// Gets the Patch version number
+    pub fn patch(&self) -> u64 {
+        self.patch
+    }
+
+    /// Gets the Pre version information
+    pub fn pre(&self) -> &MetaIdentifier {
+        &self.pre
+    }
+
+    /// Sets the Pre version information
+    pub fn set_pre(&mut self, pre: MetaIdentifier) {
```

Comment:
I think it'd be more generally useful to have `pre_mut` that returns `&mut MetaIdentifier`. If users want to set a new value they can always use `mem::replace`.